pkg/application/app.go

```diff
@@ -69,7 +69,16 @@ func Load(envName, appName string) (*Application, error) {
 	if err != nil {
 		return nil, fmt.Errorf("get app dir from env %s err %w", envName, err)
 	}
-	return LoadFromFile(filepath.Join(appDir, appName+".yaml"))
+	app, err := LoadFromFile(filepath.Join(appDir, appName+".yaml"))
+
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, fmt.Errorf(`application "%s" not found`, appName)
+		}
+		return nil, err
+	}
+
+	return app, nil
```

Comment:
It's better to raise the exception in its callers. When in creating, it should not return any error, when in `vela show`, it should.